### PR TITLE
Refresh instructions for Windows

### PR DESCRIPTION
**Pull request by simondeziel**

### CODE REVIEW

File: doc/tutorials/use.md [hunk 1]
```diff
@@ -159,7 +159,7 @@ lxc-start -n myContainerImage
...
 To make the installation a bit easier, `lxd-imagebuilder` added the `repack-windows` command. It takes a Windows ISO, and repacks it together with the necessary drivers.
 
-Currently, `lxd-imagebuilder` supports Windows 10, Windows Server 2012, Windows Server 2016, Windows Server 2019 and Windows Server 2022. The Windows version will automatically be detected, but in case this fails you can use the `--windows-version` flag to set it manually. It supports the values `w10`, `2k12`, `2k16`, `2k19` and `2k22` for Windows 10, Windows Server 2012, Windows Server 2016, Windows Server 2019 and Windows Server 2022 respectively.
+Currently, `lxd-imagebuilder` supports Windows 10, Windows 11, Windows Server 2012, Windows Server 2016, Windows Server 2019 and Windows Server 2022. The Windows version will automatically be detected, but in case this fails you can use the `--windows-version` flag to set it manually. It supports the values `w10`, `w11`, `2k12`, `2k16`, `2k19` and `2k22` for Windows 10, Windows 11, Windows Server 2012, Windows Server 2016, Windows Server 2019 and Windows Server 2022 respectively.
```
Comment (MusicDin) [inline doc/tutorials/use.md:162]: Thanks for the improved docs. I went with the approach to have a table of supported versions, as it is a bit easier to read, maybe can serve as some inspiration: https://github.com/canonical/lxd-imagebuilder/pull/86/commits/d3c07039ccd344f15ec5b2717bff45541a77aab6

Comment (simondeziel) [reply doc/tutorials/use.md:162]: Ah great, thanks for bringing support (via cherry-picks) for those EOL Windows versions. Indeed, your table's better and I'll simply rework the paragraph to mention it is autodetected but can be forced if needed.

File: doc/tutorials/use.md [hunk 2]
```diff
@@ -159,6 +159,14 @@ lxc-start -n myContainerImage
 With LXD it's possible to run Windows VMs. All you need is a Windows ISO and a bunch of drivers.
 To make the installation a bit easier, `lxd-imagebuilder` added the `repack-windows` command. It takes a Windows ISO, and repacks it together with the necessary drivers.
 
+````{tip}
+When repacking a Windows ISO, `lxd-imagebuilder` uses external tools that may need to be installed. On a Ubuntu/Debian system, those can be installed with:
+
+```bash
+sudo apt-get install -y genisoimage libwin-hivex-perl rsync wimtools
```
Comment (tomponline) [inline doc/tutorials/use.md:166]: --no-install-recommends?

**Review by tomponline — APPROVED** [review]

**Comment by tomponline** [comment]
ready for rebase

**Review by MusicDin — DISMISSED** [review]

**Review by tomponline — APPROVED** [review]

**Comment by simondeziel** [comment]
@ru-fu I should have notified you for this PR. Happy to address any feedback you might have in a follow-up PR, thanks!

**Comment by ru-fu** [comment]
> @ru-fu I should have notified you for this PR. Happy to address any feedback you might have in a follow-up PR, thanks!

No problem at all. You should just check and fix the output :wink: : https://canonical-lxd-imagebuilder.readthedocs-hosted.com/en/latest/tutorials/use/#repack-windows-iso

**Comment by simondeziel** [comment]
Thanks @ru-fu, #88 should fix the formatting.

**Pull request merged**